### ISSUE TITLE
Cleanup piwik analytics deployment

### ISSUE THREAD

**Issue by darkk** [ooni/sysadmin#149]
After #127:
- [ ] use explicit piwik version during deploy
- [ ] use `tar` instead of `zip` during deploy
- [ ] re-enable `letsencrypt` role
- [ ] cleanup variables with same name defined in different roles (e.g. `mysql_host`, `server_name`)
- [ ] it's unclear if pgp5-fpm process should have www-data gid to listen on socket.
- [ ] define "default" set of protocols / cyphers / DH params and so on for all nginx instances
- [ ] `/srv/piwik/releases/*/piwik` should be mostly root-owned
- [ ] monitor `/ping` page
- [ ] **fatal**: `expires max` on file that is changed on update makes it cached in browsers forever, re-deploy piwik on different domain to fix that

**Comment by hellais** [comment]
> fatal: expires max on file that is changed on update makes it cached in browsers forever, re-deploy piwik on different domain to fix that

I would not classify this as fatal, nor would I require re-deploy on different domain. If we must expire the file, we can just add a cache buster to the end of the request (ex. `/piwik.js?RANDOM_NONCE`)

**Comment by hellais** [comment]
I did some work on this in this PR: https://github.com/ooni/sysadmin/pull/228.

I believe of the items you list these are done:
* use explicit piwik version during deploy
* re-enable letsencrypt role
* monitor /ping page <- I have enabled support for this
* expires max on file that is changed on update ...

Most of it was done by virtue of the fact of vendoring the latest piwik nginx configuration from: https://github.com/matomo-org/matomo-nginx/commit/389a67099abf94e4df222826be6a3b06c0f4ebc1.
This commit contains the differences compared to that current master branch: https://github.com/ooni/sysadmin/pull/228/commits/9c9b8cf5a592ada6356c48cef1669ca38010746b.

I didn't do these though:
> * use tar instead of zip during deploy 
This doesn't seem like a big problem. Is there a specific reason you would like to do this?

> define "default" set of protocols / cyphers / DH params and so on for all nginx instances

I guess this should probably be done as part of some "base" ansible configuration settings

> cleanup variables with same name defined in different roles

I was not entirely sure what you meant by this

> /srv/piwik/releases/*/piwik should be mostly root-owned
> it's unclear if pgp5-fpm process should have www-data gid to listen on socket

I didn't check these, but we should try this out when deploying to ensure that nothing breaks.

As this turned out to be a bit more involved than I was originally expecting it would be useful to get some extra eyes on this.
